src/components/home/Contact.js

```diff
@@ -28,6 +30,7 @@ class Contact extends Component {
     this.setSectionRef = this.setSectionRef.bind(this);
     this.setSchoolFieldRef = this.setSchoolFieldRef.bind(this);
     this.setEmailFieldRef = this.setEmailFieldRef.bind(this);
+    this.formValid = this.formValid.bind(this);
```

Comment:
Nit: This seems to be adding code in a commit with the message "adding comments"